paymentsheet/res/drawable/stripe_link_logo_knockout.xml

```diff
@@ -0,0 +1,25 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Needed to add a new icon with no hardcoded `alpha` so that tinting works properly.